packages/payment-processor/src/payment/batch-conversion-proxy.ts

```diff
@@ -32,33 +40,45 @@ import { CurrencyInput, isERC20Currency, isISO4217Currency } from '@requestnetwo
  * @param enrichedRequests List of EnrichedRequests to pay
  * @param version The version of the batch conversion proxy
  * @param signerOrProvider The Web3 provider, or signer. Defaults to window.ethereum.
+ * @param skipFeeUSDLimit Setting the value to true skips the USD fee limit, and reduce gas consumption.
+ *                        It can be useful to set it to false if the total amount of the batch is important.
+ *                        Check the value of batchFeeAmountUSDLimit of the batch proxy deployed.
  * @param overrides Optionally, override default transaction values, like gas.
- * @dev We only implement batchRouter using two ERC20 functions:
+ * @dev We only implement batchPayments using two ERC20 functions:
  *      batchMultiERC20ConversionPayments, and batchMultiERC20Payments.
  */
 export async function payBatchConversionProxyRequest(
   enrichedRequests: EnrichedRequest[],
   version: string,
   signerOrProvider: providers.Provider | Signer = getProvider(),
+  skipFeeUSDLimit = true,
```

Comment:
I can but is it a bit early?
As I have only this argument now, and we don't if/when we are gonna update batch payments.

Reply:
imo adopting this style for all methods would allow to have always the same arguments:

pay(request, version, signer, options) 

where options vary depending on the method 

not a blocker, your call

Reply:
I am almost fine with your suggestion: pay(request, signer, options, overrides), or maybe also overrides within options?
But some functions look like pay(request, signer, amount, feeAmount, overrides), or even pay(request, signer, paymentSettings, amount, feeAmount, overrides).

I am down to do if it becomes a kind of pay() convention within RN